### DISCUSSION TITLE
Repositories are not visible.

### DISCUSSION THREAD

**Discussion by Gurkirat900** [community/community#74471] [New to GitHub]
I had made some repositories which were being shown on my account few days back but now when i search for them it shows no such repositories found. Why is that and where can I find them?

**Comment by queenofcorgis** [comment]
Hi there - do you mean when you search for them in GitHub or when you are trying to work on them with Git or your programming language?

**Reply by Gurkirat900** [reply]
Hey, No i cant see them on my github but i do have them on my desktop

**Comment by MegatIrfan** [comment]
Check the visibility settings of your repositories. If a repository is set to private, it won't be visible in search results for users who don't have access to it. Make sure you are searching for repositories within the correct visibility

**Comment by SuperZekes** [comment]
Ok I dont know for sure if this is the issue but first click on your profile picture on the top right. Then click the your profile button next click the repositories button which is next to overview at the top left. You can now see all of your repositories as well as private ones I hope this works for you.
